tests/unit_tests/c_wrappers/res/job_queue/test_ext_job.py

```diff
@@ -79,54 +81,32 @@ def test_load_forward_model_upgraded():
 def test_load_forward_model_missing_raises():
     with open("CONFIG", "w") as f:
         f.write("EXECUTABLE missing_script.sh\n")
-    with pytest.raises(ValueError):
-        _ = ExtJob("CONFIG", True)
+    with pytest.raises(ConfigValidationError, match="Could not find executable"):
+        _ = ExtJob.from_config_file("CONFIG")
 
 
 @pytest.mark.usefixtures("use_tmpdir")
 def test_load_forward_model_execu_missing_raises():
     with open("CONFIG", "w") as f:
         f.write("EXECU missing_script.sh\n")
-    with pytest.raises(ValueError):
-        _ = ExtJob("CONFIG", True)
+    with pytest.raises(ConfigValidationError, match="CONFIG failed"):
```

Comment:
can we match more on the error message? or is this just a generic failure?

Reply:
This is a generic failure for now. We will have to replace/improve the config parser to get better error messages.

Reply:
alrighty, cool, thanks!